### ISSUE TITLE
Alternative cloaking device description

### ISSUE THREAD

**Issue by jafdy** [endless-sky/endless-sky#838]
It may be a bit long (238 words) and grammar is not my strong point. The second paragraph is a bit more reasonable (124 words).

The ability to completely cloak a spaceship has for centuries baffled the best scientists and engineers humanity has to offer. The problem was primarily the need to somehow stop the inferred radiation emitted by the ship, the only obvious way of doing this is to cool the hull of the ship to about the temperature of the cosmic microwave background. The main problem with that is that all that heat must go somewhere so the only IR cloak prototype ever made was only able to cloak one side of the test ship and at the cost of lighting the other side up like a beacon and it was still picked up on active sensors.

The alien device that you picked up seems to be designed around a pair of twin matter/antimatter micro singularities that fold space-time around your ship so that any particles falling towards the ship will fall directly into the singularities. The field around your ship, like all other black holes, also emits small amounts of IR consistent with background radiation. Of course no system is perfect and the singularities must be maintained somehow, this is achieved by creating gravitational wave forms from the combustion of Hyperspace fuel, cut of the supply and the singularities will safely collapse and self-annihilate. The only signature of your ship that it fails to cloak is your gravitational signature witch is amplified instead (something about the use of black holes).

**Comment by psychiclegume** [comment]
That would make a lot more sense than "Where is it?" (paraphrased).
I think that inferred is spelled infrared.

The alien device that you picked up seems to be designed around a pair of twin matter/antimatter miniature singularities that fold space-time around your ship so that any particles falling towards the ship will fall directly into the singularities. The field around your ship, like all other black holes, also emits small amounts of IR consistent with background radiation. Of course no system is perfect, and the singularities must be maintained somehow, this seems to be achieved by creating gravitational wave forms from the combustion of Hyperspace fuel. If you cut of the supply the singularities will safely collapse and self-annihilate. The only signature of your ship that it fails to cloak is your gravitational signature which is amplified instead (something about the use of black holes).

Some grammar edits

**Comment by endless-sky** [comment]
I dunno, I thought the existing description was funny. :)

Also, I feel like if the description is changed, I'd also need an outfit sprite to go with it.

**Comment by jafdy** [comment]
I'm glad you like it. 
Inferred is spelled like that as it is simply 'infer' and 'red' smashed together by physicists.

Yes I thought the original was funny as well but I always wanted to know how it works so I figgered this out and when I was done it seemed daft not to share it. 
I guess you could put them together some how maybe have 'Knowing this isn't helping you find it though.' on the end.

Interestingly the science in the first part is entirely possible and the second part has alot of theoretical ticks on it.

**Comment by Wrzlprnft** [comment]
maybe let a event change the description once the player...nah, multiple saves :/
on the other side there could be just outfit with an without a sprite and descriptions whih are exchanged once you learn about it.

for a sprite, i could give it a try, if we need one and michael has no inspiration.

PS: i like it the way it is though

**Comment by natangold** [comment]
> Inferred is spelled like that as it is simply 'infer' and 'red' smashed together by physicists.

"Infer" means "to deduce; to logically, but indirectly, conclude", whereas the prefix "infra-" means "below". Infrared radiation is "infrared" because its periodicity is just below that of the color red in the visible spectrum (i.e., it has a lower frequency and thus longer wavelength). If you see a metallic object glowing red/orange, you can be reasonably certain that it is extremely hot - you can infer the infrared.

Sorry, I used to be a teacher.

**Comment by jmhorjus** [comment]
I laughed out loud the first time I got a cloaking device and saw the current description and blank sprite. I'd be a bit sad to see it go. 

I do like this description as well though, from a lore/science perspective. Maybe since it's a bit too long for a item description anyway it could be incorporated elsewhere? Right now there's no possible way to get more than one cloaking device in the game, so perhaps this is a new mission content opportunity. :)
